v1/src/main/java/com/google/cloud/teleport/bigtable/AvroToBigtable.java

```diff
@@ -114,7 +114,7 @@ public interface Options extends PipelineOptions {
         groupName = "Source",
         description = "Input Cloud Storage File(s)",
         helpText = "The Cloud Storage path pattern where data is located.",
-        example = "gs://<BUCKET>/<FOLDER>/<PREFIX>*")
+        example = "gs://<BUCKET_NAME>/FOLDER/PREFIX*")
```

Comment:
Was the removal of the <> signs done on purpose?

Reply:
Yes, removed it intentionally because the text in <> is getting truncated in readme files.
Those variables should be set as <FOLDER_NAME> (two words with underscore) etc..